### PR TITLE
minor README changes

### PR DESCRIPTION
**Pull request by ayush9pandey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/BuildACell/BioCRNPyler/master?filepath=%2Fexamples%2F)
 
-BioCRNPyler is a Python package for the creation, manipulation,
+BioCRNPyler (pronounced Bio-Compiler) is a Python package for the creation, manipulation,
 and study of the structure, dynamics, and functions
 of complex biochemical networks.
 
-- **Website:** http://buildacell.io/BioCRNPyler
-- **Mailing list:** TBA
+- **Website:** https://github.com/BuildACell/BioCRNPyler
```
Comment (zoltuz) [inline README.md:13]: don't change this.

Comment (ayush9pandey) [reply README.md:13]: mailing list?

Comment (zoltuz) [reply README.md:13]: http://buildacell.io/BioCRNPyler

File: README.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
+- **Mailing list:** [SBTools Google Group](https://groups.google.com/g/sbtools/) Email: sbtools@googlegroups.com
 - **Source:** https://github.com/BuildACell/BioCRNPyler
 - **Bug reports:** https://github.com/BuildACell/BioCRNPyler/issues
+- **Documentation** [biocrnpyler.readthedocs.io](https://readthedocs.org/projects/biocrnpyler/)
```
Comment (ayush9pandey) [inline README.md:17]: @zoltuz  should i remove the documentation link?

**Pull request merged**